README.rst

```diff
@@ -0,0 +1,34 @@
+What is Kolibri?
+----------------
+
+Kolibri is an adaptable measured learning environment. A user can install Kolibri and serve the app on a local network,
```

Comment:
If we had to categorize it, I think it would be the closest category -- except it's a combo LMS/app, too.